lib/ui/views/signup/signup_view.dart

```diff
@@ -32,142 +31,146 @@ class SignupView extends StackedView<SignupViewModel> with $SignupView {
-                validator: Validation.validateField,
-                decoration: InputDecoration(
-                  labelText: S.current.password,
-                  hintText: S.current.enter_your_password,
-                  suffixIcon: IconButton(
-                    onPressed: viewModel.togglePassword,
-                    icon: Icon(
-                      viewModel.hidePassword
-                          ? Icons.visibility
-                          : Icons.visibility_off,
+                  validator: Validation.validateField,
+                  decoration: InputDecoration(
+                    labelText: S.current.password,
+                    hintText: S.current.enter_your_password,
+                    suffixIcon: IconButton(
+                      onPressed: viewModel.togglePassword,
+                      icon: Icon(
+                        viewModel.hidePassword
+                            ? Icons.visibility
+                            : Icons.visibility_off,
+                      ),
                     ),
                   ),
                 ),
-              ),
-              SizedBox(
-                height: 129.h,
-              ),
-              PrimaryButton(
-                buttonText: S.current.sign_up,
-                onTap: () {
-                  if (_signupKey.currentState?.validate() ?? false) {
-                    viewModel.signup();
-                  }
-                },
-              ),
-              SizedBox(
-                height: 16.h,
-              ),
-              Text.rich(TextSpan(
-                  text: S.current.already_have_an_account,
-                  style: typography?.titleRegular16
-                      ?.copyWith(color: palette?.gray8, fontSize: 14.sp),
-                  children: [
-                    const TextSpan(text: " "),
-                    TextSpan(
-                        text: S.current.login,
-                        style: typography?.titleBold16?.copyWith(
-                          color: palette?.primary6,
-                          fontSize: 14.sp,
-                        ),
-                        recognizer: TapGestureRecognizer()
-                          ..onTap = viewModel.actionRouteToLoginView)
-                  ]))
-            ],
+                SizedBox(
+                  height: 129.h,
+                ),
+                PrimaryButton(
+                  buttonText: S.current.sign_up,
+                  onTap: () {
+                    if (_signupKey.currentState?.validate() ?? false) {
+                      viewModel.signup();
+                    }
+                  },
+                ),
+                SizedBox(
+                  height: 16.h,
+                ),
+                Text.rich(TextSpan(
+                    text: S.current.already_have_an_account,
+                    style: typography?.titleRegular16
+                        ?.copyWith(color: palette?.gray8, fontSize: 14.sp),
```

Comment:
sorry, I am just seeing this. We don't modify fontSize please

Reply:
okay